packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -2,9 +2,31 @@ import * as AdaptiveCard from 'adaptivecards';
 import MDTextInput from '@ciscospark/react-component-md-text-input';
 import MDToggleInput from '@ciscospark/react-component-md-toggle-input';
 import MDChoiceSetInput from '@ciscospark/react-component-md-choiceset-input';
+import {parseMarkdown} from '@ciscospark/redux-module-activity';
 
 import {CARD_ATTACHMENT_TYPE, TEXT_INPUT_ELEMENT, TOGGLE_INPUT_ELEMENT, CHOICE_SET_INPUT_ELEMENT} from './constants';
 
+/* eslint no-param-reassign: 2 */
+// eslint-disable-next-line valid-jsdoc
```

Comment:
If you add the docs for the `result` param, you won't need to disable the eslint.

Reply:
done